modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+
+[id="cluster-logging-deploy-multitenant_{context}"]
+= Installing cluster logging into a multitenant network
+
+If you are deploying cluster logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+
+Because the Elasticsearch Operator and the Cluster Logging Operator are installed in different projects, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. How you allow this access depends on how you configured multitenant isolation mode.
+
+* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, use the following command to join the two projects:
```

Comment:
Can this be turned into a 2-step (ordered or unordered, since I suppose they're mutually exclusive) `.Procedure`?